website_and_docs/content/documentation/grid/advanced_features/customize_node.en.md

```diff
@@ -3,13 +3,11 @@ title: "Customizing a Node"
 linkTitle: "Customize Node"
 weight: 4
 aliases: [
-"/documentation/en/grid/grid_4/customize_node/",
-"/documentation/grid/advanced_features/customize_node/"
 ]
```

Comment:
The whole `aliases` section should be removed.

Reply:
Fixed and pushed the changes. Please check @diemol